src/main/resources/assets/utilitiesinexcess/lang/en_US.lang

```diff
@@ -77,3 +77,15 @@ tile.cursed_earth.name=Cursed Earth
 tile.cursed_earth.desc=Spawns mobs aggressively
 
 utilitiesinexcess.config.block.cursed_earth=Cursed Earth Configuration
+
+tile.sound_muffler.name=Sound Muffler
+tile.sound_muffler.desc.1=Muffles sounds within %s blocks to %s%% volume
+tile.sound_muffler.desc.2=Supply with a redstone signal to turn off
+
+tile.rain_muffler.name=Rain Muffler
+tile.rain_muffler.desc.1=Silences weather sounds %s blocks
+tile.rain_muffler.desc.2=Supply with a redstone signal to turn off
+tile.rain_muffler.desc.3=Right click to toggle weather sounds globally
+tile.rain_muffler.global.enable=You dull your senses to the sound of the rain
+tile.rain_muffler.global.disable=You become aware of the sound of the rain
```

Comment:
Maybe rename to .chat.global_enable / disable? could make it clearer what it means